### PR TITLE
feat: Accept JSON string as `metadata` in browse transactions endpoint [TA-2131]

### PR DESCRIPTION
**Pull request by szeying-lee**
# Description
Updates the example and description of `metadata` as it now only accepts a JSON formatted string.

Fixes # (issue)
[TA-2131](https://gr4vy.atlassian.net/browse/TA-2131)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: reference/query-parameters/list_transactions__metadata.yaml [hunk 1]
```diff
@@ -3,15 +3,14 @@ schema:
...
+  example: ['{"key": "value"}', '{"key_one": "value", "key_two": "value"}']
 explode: true
 in: query
 name: metadata
 description: |-
   Filters for transactions where their `metadata` values
-  contain all of the provided `metadata` keys. When querying
-  for more than one key-value pair, each pair must be
-  separated by a `|` character.
+  contain all of the provided `metadata` keys. The value you send
```
Comment (hasier) [inline reference/query-parameters/list_transactions__metadata.yaml:12]: [minor] `The value sent`

File: reference/query-parameters/list_transactions__metadata.yaml [hunk 2]
```diff
@@ -3,15 +3,15 @@ schema:
...
 in: query
 name: metadata
 description: |-
   Filters for transactions where their `metadata` values
-  contain all of the provided `metadata` keys. When querying
-  for more than one key-value pair, each pair must be
-  separated by a `|` character.
+  contain all of the provided `metadata` keys. The value sent
+  for `metadata` must be formatted as a JSON string, and all
+  keys and values must be strings.
```
Comment (hasier) [inline reference/query-parameters/list_transactions__metadata.yaml:14]: Should we add something about URLencoding the values? I guess it's obvious, and probably we don't do it for other endpoints and values, so maybe there's no need?

**Review by hasier — APPROVED** [review]
metadata

**Review by federicojasson — DISMISSED** [review]

File: reference/query-parameters/list_transactions__metadata.yaml (hunk 2 above)
Comment (szeying-lee) [reply reference/query-parameters/list_transactions__metadata.yaml]: Added in https://github.com/gr4vy/gr4vy-openapi/pull/186/commits/20983372917e6b18050f1c753be04d69bf4e1c21, let me know if this is OK.

**Review by federicojasson — APPROVED** [review]

**Pull request merged**